### PR TITLE
Fix Bug 1440368, remove font observer lib, replace with font-display

### PR DESCRIPTION
**Pull request by schalkneethling**
This removes the `FontFaceObserver` library as well as related files from the codebase. Font loading is now handled using `@font-face` specifying the font loading behaviour using `font-display`

### CODE REVIEW

**Comment by schalkneethling** [comment]
@hobinjk r?

**Comment by schalkneethling** [comment]
Thanks for the review @hobinjk - I did have it but, seeing those are local fonts that is present on the users system, I do not foresee a need for those.

**Comment by hobinjk** [comment]
Good point! Thanks for the clarification

File: kuma/static/js/libs/fontfaceobserver/fontfaceobserver.2.0.7.js [hunk 1]
```diff
@@ -1,11 +0,0 @@
-(function(){'use strict';var f,g=[];function l(a){g.push(a);1==g.length&&f()}function m(){for(;g.length;)g[0](),g.shift()}f=function(){setTimeout(m)};function n(a){this.a=p;this.b=void 0;this.f=[];var b=this;try{a(function(a){q(b,a)},function(a){r(b,a)})}catch(c){r(b,c)}}var p=2;function t(a){return new n(function(b,c){c(a)})}function u(a){return new n(function(b){b(a)})}function q(a,b){if(a.a==p){if(b==a)throw new TypeError;var c=!1;try{var d=b&&b.then;if(null!=b&&"object"==typeof b&&"function"==typeof d){d.call(b,function(b){c||q(a,b);c=!0},function(b){c||r(a,b);c=!0});return}}catch(e){c||r(a,e);return}a.a=0;a.b=b;v(a)}}
```
[the remark this answers is not in the document]
Comment (schalkneethling) [reply kuma/static/js/libs/fontfaceobserver/fontfaceobserver.2.0.7.js]: This is the lib we used to rely on to load font async. As we are no longer using it, we can safely remove.

**Comment by schalkneethling** [comment]
> I'm curious about the font-awesome.styl change

Cleanup, we do not use Stylus. Perhaps we did at some point in the past or, it is part of the bundle one downloads when grabbing FontAwesome.

Thx for the review @jwhitlock

File: kuma/static/js/libs/fontfaceobserver/fontfaceobserver.2.0.7.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jwhitlock) [reply kuma/static/js/libs/fontfaceobserver/fontfaceobserver.2.0.7.js]: Oops, I meant to flag the ``font-awesome.styl`` removal with this comment. Sorry for the confusion, and thanks for [addressing that question](https://github.com/mozilla/kuma/pull/4699#issuecomment-371716153) 👍.

**Pull request merged**